app/components/container/containerController.js

```diff
@@ -117,22 +117,28 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
   };
 
   $scope.confirmRemove = function () {
+    var title = 'You are about to remove a container.';
     if ($scope.container.State.Running) {
-      ModalService.confirmDeletion(
-        'You are about to remove a running container.',
-        function (confirmed) {
-          if(!confirmed) { return; }
-          $scope.remove();
-        }
-      );
-    } else {
-      $scope.remove();
+      title = 'You are about to remove a running container.';
     }
+    var text = 'Automatically remove non-persistent volumes<i></i>';
```

Comment:
Now that I think about it, why is the text declared here and in containersController.js line 246? This could be centralized directly in the `service.confirmContainerDeletion` method in modalService.js.